Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -103,20 +109,22 @@ extension AssetsCatalogParser {
     guard item.isDirectory else { return nil }
     let type = item.extension ?? ""
 
-    switch (type, AssetCatalog.Item.supported.contains(type)) {
-    case (AssetCatalog.Item.imageSet, _):
+    switch AssetCatalog.Item(rawValue: type) {
+    case .colorSet?:
+      let name = item.lastComponentWithoutExtension
+      return .color(name: name, value: "\(prefix)\(name)")
+    case .imageSet?:
       let name = item.lastComponentWithoutExtension
       return .image(name: name, value: "\(prefix)\(name)")
-    case ("", _), (_, true):
+    default:
```

Comment:
Avoid `default`, prefer `case nil:` instead. The whole point of having an `enum` with only the supported `cases` is also to be sure that if we add a new supported type later the compiler will warn about the places where we forgot to update the code, and `default` will defeat that purpose 😉